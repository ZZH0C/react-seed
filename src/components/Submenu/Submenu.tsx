import React from 'react';
type MainSectionSubmenuProps = React.HTMLAttributes<HTMLElement>;

export const Submenu: React.FC<MainSectionSubmenuProps> = ({ children }) => {
  return <aside className="submenu">{children}</aside>;
};
