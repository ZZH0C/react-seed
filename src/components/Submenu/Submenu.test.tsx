import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
import { Submenu } from './Submenu';

describe('components/ButtonLink', () => {
  const mockProps = {
    children: 'SOME_TEXT',
  };

  beforeEach(() => {
    jest.clearAllMocks();
  });

  it('should render and match snapshot', () => {
    expect(shallow(<Submenu>{mockProps.children}</Submenu>)).toMatchSnapshot();
  });
});
