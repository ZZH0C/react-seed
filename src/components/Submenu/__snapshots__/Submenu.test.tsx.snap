// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/ButtonLink should render and match snapshot 1`] = `
<aside
  className="submenu"
>
  SOME_TEXT
</aside>
`;
